bots/images/scripts/rhel.setup

```diff
@@ -160,6 +160,8 @@ if [ "$IMAGE" = "rhel-x" ]; then
     # some packages are still missing from RHEL 8
     # HACK: https://bugzilla.redhat.com/show_bug.cgi?id=1567740
     COCKPIT_DEPS="${COCKPIT_DEPS/atomic-openshift-clients /}"
+    # docker is not a rhel-x thing (https://bugzilla.redhat.com/show_bug.cgi?id=1602951)
+    COCKPIT_DEPS="${COCKPIT_DEPS/docker /}"
```

Comment:
Yeah, maybe. I did it this way around as like that it's easier to confirm that the other images are not affected -- otherwise we'd have to rebuild all rhel*  ones. It's also more symmetric  to the other excludes. Thirdly, I'd like to have explicit excludes with links to why they don't work, instead of explicit includes -- we might just forget to re-enable an include for the next rhel image then.